src/data/schema/brand.js

```diff
@@ -10,6 +10,13 @@ export const types = `
   }
```

Comment:
Remove this file and get latest changes from feature-graphql-mutations branch. We have brand schema already